core/templates/domain/collection/search-explorations-backend-api.service.spec.ts

```diff
@@ -47,6 +50,11 @@ describe('Exploration search backend API service', () => {
       let failHandler = jasmine.createSpy('fail');
       let query = escape(btoa('three'));
 
+      var explorationMetadataObject =
```

Comment:
Wrap in parenthesis when you have statements spanning multiple lines (see coding style [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
Thanks for the feedback! Made the changes.